vortoMappingEngine.js

```diff
@@ -1,82 +1,86 @@
 // Load dependencies
-var convertToXML = require("xml-js");
-var log = require("loglevel");
-var fontoxpath = require("fontoxpath"),
-    parser = require("slimdom-sax-parser");
+const convertToXML = require("xml-js");
+const log = require("loglevel");
+const fontoxpath = require("fontoxpath");
+const parser = require("slimdom-sax-parser");
 
-var mappingSpec = null;
 
-exports.setLogLevel = function (message) {
-    if (["trace", "debug", "info", "warn", "error"].includes(message)) {
-        log.setLevel(message);
+module.exports = class VortoMapper {
```

Comment:
Allows for `const VortoMapper = require("vorto-mapper");` to import the class.

We can then do a
```js
const mapper = new VortoMapper() // pass spec here or later
mapper.setMappingSpec(...)
mapper.registerCustomXPathFunction(...)
```